bindings/c/test/mako/ddsketch.hpp

```diff
@@ -89,6 +90,11 @@ class DDSketchBase {
 		} else {
 			int index = static_cast<Impl*>(this)->getIndex(sample);
 			assert(index >= 0 && index < int(buckets.size()));
+			if (buckets[index] == std::numeric_limits<uint32_t>::max()) {
+				std::cerr << "Terminating. "
+				          << "Bucket " << index << " has hit maximum threshold in DDSketch.";
```

Comment:
- Index is just a number right? This message isn't very useful to humans if so. Can you map back to operations and alatency? Like, "grvg7ui op at 10us bucket has hit maximum size" or something along those lines?
- Add information about how the user can fix this situation, i.e. adjusting `--sampling`

Reply:
DDSketch doesn't have information about the args that mako was run with. I think it would be a little awkward to change it like that. I'll add more logging about increasing sample size and maybe just put a message saying we've hit the maximum amount of stats to be collected?